src/main/java/utils/storage/json/JsonStorage.java

```diff
@@ -63,13 +239,35 @@ public CardList load() throws InkaException {
         } catch (NullPointerException | JsonSyntaxException e) {
             String absolutePath = this.saveFile.getAbsolutePath();
             logger.log(Level.WARNING, "Corrupted save file: " + absolutePath, e);
-
+            useBackup = true;
             throw new StorageCorrupted(absolutePath);
         }
 
+        if (useBackup == true) {
+            logger.log(Level.INFO, "Trying to load backup file");
+            try{  FileReader fileReader = new FileReader(backupFile);
```

Comment:
Can replace this with a [try-with-resource](https://www.baeldung.com/java-try-with-resources) so you don't have to explictly close `fileReader` or `bufferedReader`